test/utils/not-function-types.mjs

```diff
@@ -33,7 +33,7 @@ const notFunctionTypes = [
 	'~ fn',
 	'typeof fn',
 	'void fn',
-	'delete fn',
+	// 'delete fn',
```

Comment:
Intentional?

Reply:
Yah, it need non-strict mode to run test, no one uses `array.reduce(delete foo)`, not worth add a seperate test.

Reply:
Added a comment

Reply:
I can use delete property to test :)